starlite/utils/serialization.py

```diff
@@ -31,22 +34,36 @@
     # pydantic specific types
     BaseModel: lambda m: m.dict(),
     ByteSize: lambda b: b.real,
-    EmailStr: str,
     NameEmail: str,
     Color: str,
-    AnyUrl: str,
     SecretField: str,
-    ConstrainedInt: int,
```

Comment:
We don't need all those anymore. Since they all inherit from primitive type, they will be converted into it by the compat types at the end of this mapping.